public/swagger/index.html

```diff
@@ -1,18 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html><head><link href="/swagger/app.2d918a03c72d3128edc3.css" rel="stylesheet"></head>
```

Comment:
I think this digest will change every time you rebuild the CSS?  You might want to change how it is compiled to have a static url

Reply:
Alright